crates/eips/src/eip7742.rs

```diff
@@ -0,0 +1,31 @@
+//! Contains constants and utility functions for [EIP-7742](https://eips.ethereum.org/EIPS/eip-7742)
+
+use crate::eip4844::{fake_exponential, BLOB_TX_MIN_BLOB_GASPRICE, DATA_GAS_PER_BLOB};
+
+/// Controls the update rate of the blob base fee based on `target_blob_count`.
+pub const BLOB_BASE_FEE_UPDATE_FRACTION_PER_TARGET_BLOB: u64 = 1112825;
+
+/// Calculates the `excess_blob_gas` from the parent header's `blob_gas_used`, `excess_blob_gas` and
+/// `target_blob_count`.
+///
+/// Similar to [crate::eip4844::calc_excess_blob_gas], but derives the target blob gas from
+/// `parent_target_blob_count`.
+#[inline]
+pub const fn calc_excess_blob_gas(
```

Comment:
I'm a bit concerned that this will be easy to misuse if the functions have the same name, but since this takes an additional param this is okay imo

Reply:
those are now completely identical :/ EIP-7742 just expects a normalized excess gas

should we rename the new one?